packages/expo-updates/ios/EXUpdates.podspec

```diff
@@ -89,12 +115,14 @@ Pod::Spec.new do |s|
   s.user_target_xcconfig = {
     'HEADER_SEARCH_PATHS' => '"${PODS_CONFIGURATION_BUILD_DIR}/EXUpdates/Swift Compatibility Header"',
   }
+  s.vendor_bsdiff_src!
+  s.private_header_files = "EXUpdates/BSPatch/**/*.h"
```

Comment:
BSPatch uses it's on Bool struct which overrides swifts so we need to make the headers private